bundles/GraphicServiceOrder/Controller/GraphicServiceOrderController.php

```diff
@@ -46,12 +46,13 @@ public function createOrder(Request $request, OrderService $orderService)
             $orderService->createOrder($gsOrder, $uploadedFiles);
 
             // Go to form submitted page.
-            return $this->redirectToRoute('graphic_service_order_submitted');
+            return $this->redirectToRoute('graphic_service_order_submitted', [$gsOrder->getId()]);
         }
 
         // The initial form build.
         return $this->render('@GraphicServiceOrderBundle/createOrderForm.html.twig', [
             'form' => $form->createView(),
+            'user_email' => $orderService->getUserEmail(),
```

Comment:
It seems a little weird to ask an order service for a user email …

Reply:
You should use
```
$this->redirectToRoute('graphic_service_order_submitted', ['id' => $gsOrder->getId()])
```
here (cf. comment on `createOrderSubmitted`).

Reply:
changed